### PR TITLE
post: cli 9.1

### PR DESCRIPTION
**Pull request by cexbrayat**
TODO:

- [x] tslint 6.1
- [x] post date
- [x] description

### CODE REVIEW

File: _posts/2020-03-25-angular-cli-9.1.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+[Angular CLI 9.1.0](https://github.com/angular/angular-cli/releases/tag/v9.1.0) is out!✨
+
+If you want to upgrade to 9.1.0 without pain (or to any other version, by the way), I have created a Github project to help: [angular-cli-diff](https://github.com/cexbrayat/angular-cli-diff). Choose the version you're currently using (8.3.8 for example), and the target version (9.1.0 for example), and it gives you a diff of all files created by the CLI: [angular-cli-diff/compare/8.3.8...9.1.0](https://github.com/cexbrayat/angular-cli-diff/compare/8.3.8...9.1.0).
+It can be a great help along the official `ng update @angular/core @angular/cli` command.
+You have no excuse for staying behind anymore!
+
+Let's see what we've got in this release.
+
+## Fun with flags
```
Comment (clacote) [inline _posts/2020-03-25-angular-cli-9.1.md]: Missing emoji: 🤓

File: _posts/2020-03-25-angular-cli-9.1.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: What's new in Angular CLI 9.1?
+author: cexbrayat
+tags: ["Angular 9", "Angular 8", "Angular 7", "Angular 6", "Angular 5", "Angular", "Angular 2", "Angular 4", "Angular CLI"]
```
Comment (jnizet) [inline _posts/2020-03-25-angular-cli-9.1.md]: Maybe we should stop doin this? What does that have to do ith Angular 8, 7, etc.

File: _posts/2020-03-25-angular-cli-9.1.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+It can be a great help along the official `ng update @angular/core @angular/cli` command.
+You have no excuse for staying behind anymore!
+
+Let's see what we've got in this release.
+
+## TSLint 6.1
+
+As Angular 9.1 now supports TypeScript 3.8,
+the CLI does support it as well.
+THe CLI also supports TSLint 6.1
```
Comment (jnizet) [inline _posts/2020-03-25-angular-cli-9.1.md]: THe -> The

File: _posts/2020-03-25-angular-cli-9.1.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+(which offers the support of the new TypeScript syntaxes introduced in version 3.8).
```
Comment (jnizet) [inline _posts/2020-03-25-angular-cli-9.1.md]: which supports the new

File: _posts/2020-03-25-angular-cli-9.1.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+This is slightly unusual,
```
Comment (jnizet) [inline _posts/2020-03-25-angular-cli-9.1.md]: This is slightly unusual -> This is a bit weird

File: _posts/2020-03-25-angular-cli-9.1.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+    ng update @angular/cli --migrate-only tslint-version-6
+
+## Fun with flags 🤓
+
+### ng e2e
+
+The Protractor builder now accepts two flags `grep` and `invertGrep`
+to only run the matching spec names.
+Not the file name, the spec name,
+meaning what you defined in the `describe()` and `it()` functions:
```
Comment (jnizet) [inline _posts/2020-03-25-angular-cli-9.1.md]: meaning -> i.e.
defined -> specified

**Pull request closed without merging**